Source/DafnyLanguageServer/DafnyRequestNames.cs

```diff
@@ -5,5 +5,7 @@ public static class DafnyRequestNames {
     public const string GhostDiagnostics = "dafny/ghost/diagnostics";
     public const string VerificationStatusGutter = "dafny/verification/status/gutter";
     public const string VerificationSymbolStatus = "dafny/textDocument/symbolStatus";
+    public const string VerifySymbol = "dafny/textDocument/verifySymbol";
```

Comment:
> Could you perhaps separate the two sections so that it's clear what symbols are used to send to the server, and what symbols are sent from the server to the client?

Why do you want to see that here? Note that `public const string CounterExample = "dafny/counterExample";` is an existing request that's listed among the remaining notifications.

I'm not sure what the purpose is of having these constants in a single file. I'm guessing the goal is to have a starting point for finding all the added APIs. If we split this file into two then that feature is lost and I think we might as well move all the constants to where they are used.

Maybe we can have a separate PR where we think about how to document our LSP extensions?

Reply:
> I'm not sure what the purpose is of having these constants in a single file. I'm guessing the goal is to have a starting point for finding all the added APIs. If we split this file into two then that feature is lost and I think we might as well move all the constants to where they are used.

I did not say split into two files. Both classes can be there as well. I understand your concern that the file should match the class.
Then, at least, please separate them visually and/or with a comment.

And since you observed that counterExample is a push notification (which I didn't know), please move it to the same section visually.

E.g.

```csharp
  public static class DafnyRequestNames {
    // Notifications from server to client
    public const string CompilationStatus = "dafny/compilation/status";
    public const string GhostDiagnostics = "dafny/ghost/diagnostics";
    public const string VerificationStatusGutter = "dafny/verification/status/gutter";
    public const string VerificationSymbolStatus = "dafny/textDocument/symbolStatus";
    // Requests from client to server
    public const string CounterExample = "dafny/counterExample";
    ... your code here...
}
```